main.tf

```diff
@@ -5,6 +5,9 @@
 locals {
   kp_endpoints = { for key, value in ibm_resource_instance.key_protect_instance.extensions : key => value
   }
+
+  # tflint-ignore: terraform_unused_declarations
```

Comment:
how about bumping this module to tf 1.9, and do this validation in the variables.tf using cross variable validation feature?